### ISSUE TITLE
Error trying to change dates management

### ISSUE THREAD

**Issue by Dcronfalk** [sql-bi/Bravo#730]
### Describe the problem

Error HTTP/500: Internal Server Error.
Sequence contains no matching element

### Bravo version

1.0.5 (1.0.8623.25801)

### Tabular model

Power BI Desktop, import mode

### Power BI

2.121.903.0 64-bit (september 2023)

### Screenshots

![image](https://github.com/sql-bi/Bravo/assets/146922648/4345da40-30ec-40d3-ba17-067910050475)

### Anything else ?

Trace Id: 00-49dbf85714fa1d31ff3dc8659c001b96-6eb305245a45cecf-00

**Comment by albertospelta** [comment]
More error details we got from the telemetry

`System.InvalidOperationException: Sequence contains no matching element`

```
Call Stack: System.InvalidOperationException:
   at System.Linq.ThrowHelper.ThrowNoMatchException (System.Linq, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Linq.Enumerable.First (System.Linq, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at Dax.Template.Tables.TableTemplateBase.<RestoreAffectedRelationships>g__GetColumn|28_0 (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Tables.TableTemplateBase.RestoreAffectedRelationships (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Tables.TableTemplateBase.ApplyTemplate (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Tables.Dates.BaseDateTemplate`1.ApplyTemplate (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Tables.TableTemplateBase.ApplyTemplate (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Engine+<>c__DisplayClass5_0.<ApplyTemplates>g__ApplyHolidaysTable|3 (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Dax.Template.Engine+<>c__DisplayClass5_0.<ApplyTemplates>b__0 (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at System.Collections.Generic.List`1.ForEach (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Dax.Template.Engine.ApplyTemplates (Dax.Template, Version=0.0.0.0, Culture=neutral, PublicKeyToken=c9a600d152d69fbe)
   at Sqlbi.Bravo.Infrastructure.Services.DaxTemplate.DaxTemplateManager.GetPreviewChanges (Bravo, Version=1.0.8623.25801, Culture=neutral, PublicKeyToken=null: D:\a\1\s\src\Infrastructure\Services\DaxTemplate\DaxTemplateManager.cs:104)
   at Sqlbi.Bravo.Infrastructure.Services.DaxTemplate.DaxTemplateManager.GetPreviewChanges (Bravo, Version=1.0.8623.25801, Culture=neutral, PublicKeyToken=null: D:\a\1\s\src\Infrastructure\Services\DaxTemplate\DaxTemplateManager.cs:87)
   at Sqlbi.Bravo.Services.ManageDatesService.GetPreviewChanges (Bravo, Version=1.0.8623.25801, Culture=neutral, PublicKeyToken=null: D:\a\1\s\src\Services\ManageDatesService.cs:78)
```

**Comment by albertospelta** [comment]
@Dcronfalk we were able to identified the error from telemetry but to fix it we need to be able to reproduce the issue.
Could you share a minimal .pbix with some dummy data that shows this issue? Thanks!

**Comment by Dcronfalk** [comment]
Hi Alberto, I am not able to attach pbix files here by drag and drop. Is there another way I can send you the dummy file?

**Comment by albertospelta** [comment]
@Dcronfalk you can send it to alberto.spelta@sqlbi.com

**Comment by albertospelta** [comment]
@Dcronfalk the bug has been fixed and will be included in the next release. If you are interested in trying a preview build, you can get it by changing the update channel from Stable to Dev in the Bravo options panel and then follow the download link.

**Comment by albertospelta** [comment]
Fixed in version [1.0.6](https://github.com/sql-bi/Bravo/releases/tag/v1.0.6)
